impl/src/context.rs

```diff
@@ -15,11 +15,16 @@
 use crate::settings::CrateSettings;
 use serde::Serialize;
 
+/** A struct containing information about a crate's dependency that's buildable in Bazel 
+ * 
+ * Note: The ordering of the this struct's fields is important to ensure that the output 
```

Comment:
(I'm not sure if i'm being reasonable here, sorry if I'm not)

I think you should be as specific as possible here:

```Note: BUILDifier-compliant BUILD file generation depends on correct sorting of collections of this struct by `buildable_target`. Do not add fields before that field.```

(this is part of a general philosophy of comment and code writing I've acquired -- my north star is to find the perfect balance of terse and detailed. One useful heuristic is to start with the most specific explanation, and then decide whether that needs to be cut down. Typically, a good specific explanation happens also to be terse)

Reply:
Yeah, this is a good suggestion. I can remember to be more specific about things like this. It's definitely something that will help everyone.

Reply:
Updated the comment.